homeassistant/components/shelly/sensor.py

```diff
@@ -430,11 +430,18 @@ class RestSensorDescription(RestEntityDescription, SensorEntityDescription):
 
 def _build_block_description(entry: RegistryEntry) -> BlockSensorDescription:
     """Build description when restoring block attribute entities."""
+    native_unit_of_measurement: str | None
+    # For the temperature entity native unit is Celsius but unit_of_measurement in entry
+    # will be Farenheit if user is using that unit.
```

Comment:
`Fahrenheit` typo, and probably more readable as
```suggestion
    # For the temperature entity, the native unit is Celsius,
    # but the entry's unit_of_measurement will be Fahrenheit
    # if the user is using that unit.
```